backend/package-lock.json

```diff
@@ -21,6 +21,7 @@
         "class-transformer": "^0.5.1",
         "class-validator": "^0.13.2",
         "dotenv": "^16.0.3",
+        "nest": "^0.1.6",
```

Comment:
I'm afraid this is not related to Nest.JS, but rather this project: https://www.npmjs.com/package/nest

Better for us to remove it please